DotNet/Report/Application/Extensions/KafkaProducerRegistration.cs

```diff
@@ -0,0 +1,34 @@
+﻿using Confluent.Kafka;
+using LantanaGroup.Link.Report.Application.Models;
+using LantanaGroup.Link.Shared.Application.Factories;
+using LantanaGroup.Link.Shared.Application.Interfaces;
+using LantanaGroup.Link.Shared.Application.Models.Configs;
+
+namespace LantanaGroup.Link.Report.Application.Extensions;
+
+public static class KafkaProducerRegistration
+{
+    public static void RegisterKafkaProducers(this IServiceCollection services, KafkaConnection kafkaConnection)
+    {
+        services.AddTransient<IKafkaProducerFactory<string, DataAcquisitionRequestedValue>, KafkaProducerFactory<string, DataAcquisitionRequestedValue>>();
+        services.AddTransient<IKafkaProducerFactory<SubmissionReportKey, SubmissionReportValue>, KafkaProducerFactory<SubmissionReportKey, SubmissionReportValue>>();
+        services.AddTransient<IKafkaProducerFactory<string, string>, KafkaProducerFactory<string, string>>();
+
+        var dataAcqProducerConfig = new ProducerConfig()
+        {
+            ClientId = "Report_DataAcquisitionScheduled"
```

Comment:
Is this intended to be 'Report_DataAcquisitionRequested'?

Reply:
In the job, it shows what is above. Please refer to line 67 in GenerateDataAcquisitionRequestsForPatientsToQuery.cs